src/test/jtx/impl/Env.cpp

```diff
@@ -537,6 +537,14 @@ Env::do_rpc(std::vector<std::string> const& args)
     return response;
 }
 
+void
+Env::enableFeature(uint256 const feature)
```

Comment:
I agree, but I feel like it's a nice convenience method for testing (can we mark it as such?), especially when enabling a feature in between a transaction. @nbougalis any thoughts?